hack/gen-patch-defaultrestmapper.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
```

Comment:
Is this now part of the rebathe se procedure? Do you think we should add it to the docs?

Reply:
The script is now run as a part of `make codegen` ; that way we don't need to rely on docs.

In general it really should be in the docs too, but the rebase docs are missing a lot of things. I can open a ticket and work on updating them, and mention the need to run the various generators we have. Would that be ok?